snapshot.py

```diff
@@ -0,0 +1,44 @@
+import json
+import subprocess
+
+def establish_ssh_connectivity():
+    node_ip_address_list = []
+    command = ["oc", "get", "nodes", "-o", "json"]
+    node_output=subprocess.check_output(command)
+
+    node_map=json.loads(node_output)
+    for item in node_map["items"]:
+        if "node-role.kubernetes.io/control-plane" in item["metadata"]["labels"]:
+            node_ip_address_list.append(item["status"]["addresses"][0]["address"])
+
+    for node_ip_address in node_ip_address_list:
+        command = ["ssh", "-o", "StrictHostKeyChecking=no", f"core@{node_ip_address}", "true"]
+        print(subprocess.run(command))
```

Comment:
Why are we using print() rather than the logger the rest of the repo uses?

Reply:
@SalDaniele currently refactoring. Ignore the prints, I am replacing them with logger.